lib/bookstore-scrapper/indexTextbooks.mts

```diff
@@ -0,0 +1,51 @@
+/*
+    Run:
+    tsc --downlevelIteration --target es2021 --moduleResolution node --module es2022 --allowSyntheticDefaultImports indexTextbooks.mts; 
+    NODE_TLS_REJECT_UNAUTHORIZED='0' node indexTextbooks.mjs
+*/
+
+import { CampusStoreEntry } from './helpers/getTextbookInformation.mjs';
+import { updateDatabase } from './database/updateDatabase.mjs';
+import { getBookData } from './helpers/getBookData.mjs';
+import { axiosRequest } from './helpers/axiosRequest.mjs';
+import {
+    updateAlgoliaIndex,
+    getAlgoliaObject,
+} from './algolia/updateAlgoliaIndex.mjs';
+import fs from 'fs';
+
+// Algolia constants - FILL BEFORE RUNNING
+export const algoliaAppId: string = '';
+export const algoliaApiKey: string = '';
+export const algoliaIndexName: string = '';
```

Comment:
Can these be stored as environment variables? We will need them to be eventually for the frontend anyways.

Reply:
Sure, will do.